cmd/apiserver/apiserver.go

```diff
@@ -27,8 +27,6 @@ import (
 	"k8s.io/kubernetes/pkg/util/logs"
 
 	"github.com/kubernetes-incubator/service-catalog/cmd/apiserver/app/server"
-	// install our API groups
-	_ "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
```

Comment:
The `install` is actually called multiple times throughout the code, in several tests and also `cmd/controller_manager`, which is where our API gets installed.